psql_authnz/psql_authnz.py

```diff
@@ -30,7 +31,7 @@ def main():
 
     # Setup logging
     LOG_FORMAT = "%(asctime)-15s PSQL-AUTHNZ [%(levelname)-5s]: %(filename)-15s:%(lineno)-3s - %(message)s"
-    logging.basicConfig(format=LOG_FORMAT,level=getattr(logging, log_level.upper()))
+    logging.basicConfig(filename=log_file,format=LOG_FORMAT,level=getattr(logging, log_level.upper()))
```

Comment:
Actually, that's not a fast rule, that's just a convention.  If you looked at the source for logging.basicConfig, it probably is defined like: def basicConfig(filename=None, ...), which is why passing None does the same thing.  If it defaulted to something else, this would not be the case.